<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedyaLib</class>
 <widget class="QMainWindow" name="MedyaLib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedyaLib</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
        <item row="0" column="2" rowspan="2">
         <layout class="QGridLayout" name="layoutSearch">
          <item row="0" column="0">
           <widget class="QWidget" name="widgetDetailedSearch" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="9">
              <widget class="QCheckBox" name="checkExactMatch">
               <property name="text">
                <string>Exact Match</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="labelSrchLoc">
               <property name="text">
                <string>Location</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" rowspan="2" colspan="11">
              <layout class="QHBoxLayout" name="layoutPresentation"/>
             </item>
             <item row="0" column="8">
              <widget class="QLineEdit" name="lineDetailedTimes"/>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="lineDetailedLocation"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelSrchTags">
               <property name="text">
                <string>Tags</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineDetailedTags"/>
             </item>
             <item row="0" column="7">
              <widget class="QLabel" name="labelSrchTime">
               <property name="text">
                <string>Times</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" rowspan="3">
              <layout class="QVBoxLayout" name="layoutTree">
               <property name="spacing">
                <number>0</number>
               </property>
              </layout>
             </item>
             <item row="0" column="6">
              <widget class="QLineEdit" name="lineDetailedWho"/>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="labelSrchWho">
               <property name="text">
                <string>Who</string>
               </property>
              </widget>
             </item>
             <item row="0" column="11">
              <widget class="QToolButton" name="toolSearch">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
             <item row="0" column="10">
              <widget class="QPushButton" name="pushApplyFilters">
               <property name="text">
                <string>Apply Filters</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNewInput">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
        <item>
         <layout class="QGridLayout" name="layoutSplitter"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutNewFilters">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tags</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTags"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLocat"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Who</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWho"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Times</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTimes"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolSaveInfos">
            <property name="text">
             <string>SET</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
